src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -44,7 +32,7 @@ class FormPage extends React.Component {
         this.props.route.pageConfig.pageKey ||
       get('params.index', prevProps) !== get('params.index', this.props)
     ) {
-      scrollToTop();
+      scrollToTop('topScrollElement', window.Forms?.scroll || '');
```

Comment:
Same concern here about the empty string.

Reply:
Updated